example/src/screens/Casting.tsx

```diff
@@ -16,6 +16,11 @@ function prettyPrint(header: string, obj: any) {
 export default function Casting() {
   BitmovinCastManager.initialize();
 
+  if (Platform.OS === 'android') {
```

Comment:
Is it safe to compare using `===` in this case?
Could it be that the strings are not the same, but only equal?

Reply:
I think this is the preferred way for TypeScript. Even the linter warns about it!